dotnet/src/SemanticKernel.UnitTests/KernelTests.cs

```diff
@@ -108,13 +108,11 @@ public async Task ItUsesChatSystemPromptWhenProvidedAsync(string providedSystemC
             .WithAIService<ITextCompletion>("x", mockTextCompletion.Object)
```

Comment:
nit.

Add checks when the PromptTemplateCOnfig is misconfigured.

Models = null
Models = Array.Empty.

Reply:
I'll add more unit tests in a follow up PR. We're splitting the Semantic Function code into it's own package so these unit tests are moving.